sdks/python/setup.py

```diff
@@ -340,6 +340,7 @@ def get_portability_package_data():
       ext_modules=extensions,
       install_requires=[
           'crcmod>=1.7,<2.0',
+          'cryptography>=44.0.1',
```

Comment:
this will break beam installation on any python environment that has a dependency with upper bounded cryptography<44

To resolve vulnerability in container, update container requirements are sufficient